pom.xml

```diff
@@ -690,6 +690,11 @@
             <artifactId>commons-math3</artifactId>
             <version>3.6.1</version>
         </dependency>
+        <dependency>
+            <groupId>io.leonard</groupId>
+            <artifactId>opening-hours-evaluator</artifactId>
+            <version>1.2.1</version>
+        </dependency>
```

Comment:
I am not sure if we should have this dependency - I know it is @leonardehrenfried and @flaktack who maintain it, but I am skeptical to use a library that is just 2 classes and a thin wrapper around another lib - it is poetically creating problems in the future, while maintaining the needed logic inside OTP seem like a better option. Also the `ch.poole.openinghoursparser` lib is used in this PR directly, so we should include the dependency to it as well, and not relay on a transitive dependency. Another problem is the license - am not an expert, but we at least need to include it if we use the lib.

Reply:
We will keep using this as a depedency but we will add ch.poole.openinghoursparser to pom as an separate depedency as well.